starkbank/event/__event.py

```diff
@@ -41,7 +43,7 @@ class Event(Resource):
     list all generated updates on entities.
     ## Attributes:
     - id [string]: unique id returned when the event is created. ex: "5656565656565656"
-    - log [Log]: a Log object from one the subscription services (TransferLog, InvoiceLog, DepositLog, BoletoLog, BoletoHolmesLog, BrcodePaymentLog, BoletoPaymentLog, UtilityPaymentLog or TaxPaymentLog)
+    - log [Log]: a Log object from one the subscription services (TransferLog, InvoiceLog, DepositLog, BoletoLog, BoletoHolmesLog, BrcodePaymentLog, BoletoPaymentLog, UtilityPaymentLog, TaxPaymentLog or DarfPaymentLog)
```

Comment:
pequeno typo, o correto seria: `a Log object from one of the subscription services...`

Reply:
Alterado! Troquei pra "subscribed" também